src/main.c

```diff
@@ -119,21 +119,21 @@ int main(int argc, char *argv[])
         }
     }
     Server *server = serverNew(server_port);
-    serverAddHandler(server, notFound);
-    serverAddStaticHandler(server);
-    serverAddHandler(server, about);
-    serverAddHandler(server, signup);
-    serverAddHandler(server, logout);
-    serverAddHandler(server, login);
-    serverAddHandler(server, search);
-    serverAddHandler(server, connect);
-    serverAddHandler(server, like);
-    serverAddHandler(server, unlike);
-    serverAddHandler(server, post);
-    serverAddHandler(server, profile);
-    serverAddHandler(server, dashboard);
-    serverAddHandler(server, home);
-    serverAddHandler(server, session);
+    serverAddRoute(server, NONE, NULL, notFound);
```

Comment:
Can you think of the better way to decouple the following routing rules from `main` function? I expect the modularized URI handlers can be maintained without the hooks in `main` function.